src/routes/safe/components/AddressBook/index.jsx

```diff
@@ -160,7 +159,7 @@ const AddressBookTable = ({ classes }: Props) => {
                           className={classes.editEntryButton}
                           onClick={() => {
                             setSelectedEntry({
-                              entry: { ...row, isOwnerAddress: userOwner },
+                              entry: { ...row.toJS(), isOwnerAddress: userOwner },
```

Comment:
why `.toJS`? it's kind of an expensive operation. I would say it's better to get rid of that. Maybe something like
```
{
  owner: row,
  isOwnerAddress: userOwner
}
```

Reply:
fixed